SofaImGui/src/SofaImGui/ImGuiGUIEngine.cpp

```diff
@@ -149,7 +144,7 @@ void ImGuiGUIEngine::initBackend(GLFWwindow* glfwWindow)
     ImGui_ImplGlfw_InitForOpenGL(glfwWindow, true);
 
 #if SOFAIMGUI_FORCE_OPENGL2 == 1
-    ImGui_ImplOpenGL2_Init();
+    ImGui_ImplOpenGL3_NewFrame;
```

Comment:
I am curious about this....

Reply:
When I did the Pr the macos CI crashed with the  `ImGui_ImplOpenGL2_Init()` config and it recomended changing it to     `ImGui_ImplOpenGL3_NewFrame`

Reply:
It does not really make sense 😅
 - it is in OpenGL2 mode (hence the macro)
 - instead of using *Init(), it is doing NewFrame() ??

Reply:
Needless to say, I tried on my setup with those lines and it crashed 😶‍🌫️